src/views/console.cc

```diff
@@ -106,10 +111,196 @@ Console::LogPrivate(int level, const QString& msg)
 	m_text->insertHtml(html);
 	m_numLines++;
 	m_text->ensureCursorVisible();
+
+	if(fileLogging)
+		LogToFile(msg);
+
 	m_lock.unlock();
 }
 
 void
+Console::LogToFile(QString message)
+{
+	bool overwrite = false;
+	QSettings settings;
+	bool loggingEnabled = settings.value("mainWindow/loggingEnabled", true).toBool();
+	if(!loggingEnabled)
+		return;
+
+	QString currentTimeStamp = QDateTime::currentDateTime().toString(LOG_TIMESTAMP_FORMAT);
+	message = currentTimeStamp+": "+message;
+
+	QString filename = settings.value("mainWindow/logFileName").toString();
+	QFile* file = new QFile(filename);
```

Comment:
Just declare it on the stack so you don't have to worry about deleting it.  "QFile file(filename);"